boards/xtensa/intel_adsp_cavs18/doc/index.rst

```diff
@@ -0,0 +1,34 @@
+.. _intel_adsp_cavs18:
+
+Intel ADSP CAVS 1.8
+###################
+
+Overview
+********
+
+This board configuration is used to run Zephyr for ADSP applications on Intel
+`Whiskey Lake`_ microprocessors. It is most commonly used to run version 1.8 of
```

Comment:
cAVS1.8 ADSP is used in multiple Intel products. Whiskey Lake is one, but also present on Comet Lake and Coffee Lake.